logger/log-setup.md

```diff
@@ -0,0 +1,16 @@
+# Installation of Logbook
```

Comment:
I would list this as a requirement in requirements.txt found in the root-level directory.